modules/understanding-upgrade-channels.adoc

```diff
@@ -20,7 +20,7 @@ You can use the `candidate-{product-version}` channel to update from a previous
 
 [NOTE]
 ====
-Release candidates differ from the nightly builds. Nightly builds are available for early access to features, but updating to or from nightly builds is neither recommended nor supported. Nightly builds are not available in any upgrade channel. You can reference the {product-title}
+Release candidates differ from the nightly builds. Nightly builds are available for early access to features, but nightly builds are not supported, and updating to or from nightly builds is neither recommended nor supported. Nightly builds are not available in any upgrade channel. You can reference the {product-title}
```

Comment:
We don't offer nightly builds in any graph accessible from a default installation nor do we ever suggest they're supported. In https://github.com/openshift/openshift-docs/pull/47333/files#diff-a730a4f4ac2c41dde208df2277ff16a4411c015a889dc4ba15323ed8805df5ceL21-L26 I'm getting rid of this entirely.